doc/perf.md

```diff
@@ -245,13 +238,13 @@ $
 ```
 
 The optimizer assumes that you don't attempt to access the value of a `NULL` pointer.
-In other words, it thinks that the `x = *foo` code will never run,
-and it can therefore be ignored.
+In other words it thinks that the `x = *foo` code will never run,
+and therefore can be ignored.
 
-Correctly written Jou code does not break when optimizations are enabled.
-For example, accessing a `NULL` pointer simply isn't a good way to exit a program;
-your program shouldn't ever do it.
-If you want the program to crash, you can use the `abort()` function, for example:
+Correctly written Jou code does not break when optimizations are enabled.           ?
+For example, accessing a `NULL` pointer simply isn't a good way to exit a program;  ?
+your program shouldn't ever do it.                                                  ?
+If you want the program to crash, you can use the `abort()` function, for example:  ?
```

Comment:
What do the question marks mean? Is this part hard to understand or poorly explained?